Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumEmail.cshtml

```diff
@@ -0,0 +1,44 @@
+@page
+@attribute [RazorCompiledItemMetadata("RouteTemplate", ConsortiumStepModel.ConsortiumEmailPage)]
+@model ConsortiumEmailModel
+@using CO.CDP.Localization
+
+@{
+    ViewData["Title"] = StaticTextResource.Consortium_ConsortiumEmail_Heading;
+    var emailAddressHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.EmailAddress)).HasInnerHtml;
+    var backLink = ConsortiumStepModel.ConsortiumAddressPage;
+}
+
+@section BeforeContent {
+    <a href="@backLink" class="govuk-back-link">@StaticTextResource.Global_Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ErrorSummary" model="@Model.ModelState" />
+        <form class="form" method="post">
+            <span class="govuk-caption-l">
+                @Model.ConsortiumName
+            </span>
+            <h1 class="govuk-label-wrapper">
+                <label class="govuk-label govuk-label--l" for="consortiumName">
+                    @ViewData["Title"]
+                </label>
+            </h1>
+
+            <div class="govuk-form-group @(emailAddressHasError ? "govuk-form-group--error" : "")">
+                @if (emailAddressHasError)
+                {
+                    <p id="email-address-error" class="govuk-error-message">
+                        <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
+                        @Html.ValidationMessageFor(m => m.EmailAddress)
+                    </p>
+                }
+                <input class="govuk-input @(emailAddressHasError ? "govuk-input--error" : "")" id="@nameof(Model.EmailAddress)"
+                       value="@Model.EmailAddress" type="text" name="@nameof(Model.EmailAddress)"
+                       spellcheck="false" autocomplete="email" @(emailAddressHasError ? "aria-describedby=email-address-error" : "")>
```

Comment:
Email field type, so that on mobile the email specific keyboard will be triggered. But `novalidate` ensures the built in browser validation doesn't trigger
```suggestion
        <form class="form" method="post" novalidate>
            <span class="govuk-caption-l">
                @Model.ConsortiumName
            </span>
            <h1 class="govuk-label-wrapper">
                <label class="govuk-label govuk-label--l" for="consortiumName">
                    @ViewData["Title"]
                </label>
            </h1>

            <div class="govuk-form-group @(emailAddressHasError ? "govuk-form-group--error" : "")">
                @if (emailAddressHasError)
                {
                    <p id="email-address-error" class="govuk-error-message">
                        <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
                        @Html.ValidationMessageFor(m => m.EmailAddress)
                    </p>
                }
                <input class="govuk-input @(emailAddressHasError ? "govuk-input--error" : "")" id="@nameof(Model.EmailAddress)"
                       value="@Model.EmailAddress" type="email" name="@nameof(Model.EmailAddress)"
                       spellcheck="false" autocomplete="email" @(emailAddressHasError ? "aria-describedby=email-address-error" : "")>
```